eth/tracers/native/prestate.go

```diff
@@ -259,7 +259,7 @@ func (t *prestateTracer) GetResult() (json.RawMessage, error) {
 // Stop terminates execution of the tracer at the first opportune moment.
 func (t *prestateTracer) Stop(err error) {
 	t.reason = err
-	atomic.StoreUint32(&t.interrupt, 1)
+	t.interrupt.Store(true)
```

Comment:
I just realized through your PR that prestateTracer is ignoring the `interrupt` signal sort of. It has to check for interrupts in `CaptureState`. I'd be grateful if you add it in this PR, but don't feel obliged.